lib/config.js

```diff
@@ -14,9 +14,10 @@
  * not, you can obtain one from Tidepool Project at tidepool.org.
  * == BSD2 LICENSE ==
  */
-var pkg = require('./../package.json');
+var pkg = require('../package.json');
 
-var config = (typeof window !== 'undefined') ? window.config : {};
+//var config = (typeof window !== 'undefined') ? window.config : {};
```

Comment:
I'm not entirely sure, but also, given that there's a full node available inside Electron, the two environments should be nearly identical. Is there usefulness in differentiating given the same execution environment?